meeko/preparation.py

```diff
@@ -170,7 +170,7 @@ def __init__(
         self.dihedral_model = dihedral_model
         self.dihedral_params = dihedral_list
 
-        if dihedral_model == "espaloma" or charge_model == "espaloma":
+        if dihedral_model == "espaloma" or charge_model in ["espaloma", "zero"]:
```

Comment:
Why load espaloma for setting charges to zero?

Reply:
Espaloma is imported inside the init of EspalomaTyper to prevent loading espaloma unnecessarily, because it takes a few seconds to load, and because it is an optional dependency. So. it would be better to set the attribute to None or similar

Reply:
does "zero" mean all zeros, or total charge = zero?

Reply:
I thought "zero" still needs espaloma (or a charge method), the total charge is adjusted to zero, but maybe I'm wrong.. I

Reply:
It means all zeros. Should rename string to "all_zero" or similar.

I couldn't reproduce the attribute not defined error, but I tried only with ligand preparation. How did you get that? Was it with receptor prep?

Reply:
It means all zeros. Should rename string to "all_zero" or similar.
Got it! When I did this, I don't remember seeing charge list is set to all zero (or maybe it's initialized as all zeros? ), but I will take a closer look at the codes a little later this afternoon.

Yes, I found it with receptor prep. If I set MoleculePreparation(charge_model="gasteiger") in the script